interface/forms/fee_sheet/review/initialize_review.php

```diff
@@ -31,6 +31,7 @@
     var justify_click_title="<?php echo xls('Click to choose diagnoses to justify.')?>";
     var fee_sheet_options=[];
     var diag_code_types=<?php echo diag_code_types('json');?>;  // This is a list of diagnosis code types to present for as options in the justify dialog, for now, only "internal codes" included.
+    var ippf_specific = <?php echo $GLOBALS['ippf_specific'] ? 'true' : 'false'; ?>;
 </script>
 <script type="text/javascript" src="<?php echo $web_root;?>/library/js/jquery-1.9.1.min.js"></script>
 <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/knockout-2-2-1/build/output/knockout-latest.js"></script>
```

Comment:
Actually, when this code gets in, this will be a good place to upgrade to the newer knockout/jquery that Kevin brought in with the tab layout (and then can drop the old knockout). I'll plan to try it after your code is in the codebase.